locales/en-US/app.ftl

```diff
@@ -467,6 +468,7 @@ MenuButtons--metaInfo--name-and-version = Name and version:
 MenuButtons--metaInfo--update-channel = Update channel:
 MenuButtons--metaInfo--build-id = Build ID:
 MenuButtons--metaInfo--build-type = Build type:
+MenuButtons--metaInfo--arguments = Program arguments:
```

Comment:
What is arguments in this case?

Reply:
So, command line arguments, or something else completely? Would simply "Arguments:" work?

Reply:
Of course